app/models/plant_accession.rb

```diff
@@ -25,10 +25,25 @@ class PlantAccession < ActiveRecord::Base
 
   include Relatable
   include Filterable
-  include Pluckable
   include Searchable
   include Publishable
-  include TableData
+
+  def self.table_data(params = nil, uid = nil)
+    pv_subquery = PlantVariety.visible(uid)
+    pl_subquery = PlantLine.visible(uid)
+
+    query = all.
+      joins {[
+        pl_subquery.as('plant_lines').on { plant_lines.id == plant_accessions.plant_line_id }.outer,
+        pv_subquery.as('plant_varieties').on { plant_varieties.id == plant_accessions.plant_variety_id }.outer,
+        pv_subquery.as('plant_line_varieties').on { plant_line_varieties.id == plant_lines.plant_variety_id }.outer
```

Comment:
Note: This could potentially be changed to an inner join if every PlantLine was linked to a PlantVariety -- however a look at the production DB indicates that this is not the case. I guess it's safer to leave it as is.